src/state/apis/foxy/foxyBalancesApi.ts

```diff
@@ -220,7 +239,40 @@ export const foxyBalancesApi = createApi({
         }
       },
     }),
+    getFoxyApr: build.query<GetFoxyAprOutput, void>({
+      queryFn: async () => {
+        const response = await axios.get<{ chains: TokemakChainData[] }>(TOKEMAK_STATS_URL)
```

Comment:
If we want to be extra accurate, we should handle fetching errors and return the error code as a `{status, data}` shape